persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java

```diff
@@ -1444,7 +1444,10 @@ public void setPropertyMapping(final PropertyType property, final String itemTyp
             }
             Map<String, Object> subMappings = mappings.computeIfAbsent("properties", k -> new HashMap<>());
             Map<String, Object> subSubMappings = (Map<String, Object>) subMappings.computeIfAbsent("properties", k -> new HashMap<>());
-            mergePropertiesMapping(subSubMappings, createPropertyMapping(property));
+            Map<String, Object> propertyMapping = createPropertyMapping(property);
+            if (propertyMapping != null) {
+                mergePropertiesMapping(subSubMappings, propertyMapping);
+            }
```

Comment:
I think we should simplify this function, if `Map<String, Object> propertyMapping = createPropertyMapping(property);` is null we can simply abort this function by `return;`

Also for better perf, we can do this in the first line to avoid calculating the existing mapping in case createPropertyMapping return null.